charts/grist/examples/helmfile/values.grist.yaml

```diff
@@ -24,10 +26,16 @@ commonEnvVars: &commonEnvVars
   TYPEORM_PASSWORD: grist
   GRIST_SINGLE_PORT: 0
   GRIST_ALLOWED_HOSTS: grist.127.0.0.1.nip.io
+  GRIST_BOOT_KEY: bootkey
+  GRIST_LOG_SKIP_HTTP: false
+  NODE_OPTIONS: '--inspect --enable-source-maps' # This enables the debugger in the containers
```

Comment:
Thinking about it, I feel like the use case for debugging should be a bit different from the one for testing like in production:
 - in production-like testing, we may want to check the liveness, whereas we would like to disable it when debugging;
 - in production-like testing, we may want to test with worker replicas, while it is painful when debugging (knowing what container we should attach the debugger) and most of the times is not useful;
 - also disabling the replicas would be simpler for automating the port forwarding;

I would still keep the `GRIST_BOOT_KEY`, `GRIST_LOG_SKIP_HTTP` and `GRIST_MAX_UPLOAD_IMPORT_MB` env variables.